### ISSUE TITLE
Mutation definition doesn't accept description (v0.8.0)

### ISSUE THREAD

**Issue by kliuless** [rmosolgo/graphql-relay-ruby#32]
```ruby
GraphQL::Relay::Mutation.define { name 'TestMut'; description 'Test desc' }
# NoMethodError: undefined method `description=' for #<GraphQL::Relay::Mutation:0x007fdd7d2dbc90>
```
This seems to be related to the refactoring of the graphql gem in v0.12. Apparently Mutation instances now need to have a `description=` setter.

Update: I checked v0.7.1, and it doesn't have the error above, but it also doesn't actually expose the description for schema queries, so apparently it was just being ignored.

**Comment by rmosolgo** [comment]
Oops, you're totally right!

You can see the typo here, where I remove `:description` and add `:definition`: https://github.com/rmosolgo/graphql-relay-ruby/commit/1f4242f6be017b8d8e6b82cc55e1300df6d83469#diff-0ab056aa4abd4473823cbd7d4131d72aR57

Fixed in 834f172, plus added it to the test schema so it won't break again!

I imagine you could also work around it for the moment with 

```ruby
module GraphQL 
  module Relay 
    class Mutation 
      attr_accessor :description 
    end 
  end
end
```

**Comment by rmosolgo** [comment]
shipped in 0.8.1, please reopen if it's still a problem!

**Comment by kliuless** [comment]
Thanks, it looks good now.
